invenio_app_ils/ui/src/invenio_app_ils/common/components/SearchBar/components/QueryBuildHelper/QueryBuildHelper.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List, Grid } from 'semantic-ui-react';
+
+export default class QueryBuildHelper extends Component {
+  constructor(props) {
+    super(props);
+    this.updateQueryString = this.props.updateQueryString;
+  }
+
+  _addToQuery = field => {
+    const defaultVal = field.defaultValue ? field.defaultValue : '*';
+    const newCriteriaString = `${field.field}:${defaultVal}`;
+    const previousQueryString = this.props.currentQueryString;
+
+    if (previousQueryString === '') {
+      this.props.updateQueryString(newCriteriaString);
+    } else {
+      this.props.updateQueryString(
+        `${previousQueryString} AND ${newCriteriaString}`
+      );
+    }
+  };
+
+  _render_list_items = fields => {
```

Comment:
I see that `_addToQuery` above uses camel case while here it's snake case for the method name. Should we stick to one style?